### ISSUE TITLE
Generated audio learns first few words but not all in sentence

### ISSUE THREAD

**Issue by acrosson** [fatchord/WaveRNN#85]
Hi,

I trained tacotron on a custom dataset, about 3700 samples, about 3.8 seconds each for 676k steps.

The loss has came down to ~0.4706

The model can output words that kind of sound like words, but only the first one or two words in a sentence are understandable.

Sample output:
[samples.zip](https://github.com/fatchord/WaveRNN/files/3233995/samples.zip)

```
 tts_schedule = [(7,  1e-3,  20_000,  32),
                 (5,  1e-4, 200_000,  32),
                 (2,  1e-4, 360_000,  16),
                 (1,  1e-4, 700_000,  8)]
```

Does anyone have any suggestions?

**Comment by oytunturk** [comment]
I'm not sure 3700 sentences are sufficient to learn pronunciations from
text. You may need to include more training data.
There is some insight in this paper <https://arxiv.org/pdf/1811.06315.pdf>.
Another option might be to train using phonemes as input instead of text.
Still, it could be insufficient for the WaveRNN model.

**Comment by oytunturk** [comment]
In addition, I think multi-speaker model training is the way to go to make these models more robust when training data from a given speaker is limited. Samples and paper using a similar idea is [here](https://nii-yamagishilab.github.io/sample-tts-speaker-imbalanced/index.html).

**Comment by acrosson** [comment]
Thanks @oytunturk for the suggestions. I'm going to try training with more data and see how that works. I'll update soon
